app/pages/docs/why-blitz.mdx

```diff
@@ -1,73 +1,75 @@
+De Blitz data layer is een nieuwe technologie voor het elimineren van de API met behoud van client-side rendering. In een Blitz app kun je je servercode rechtstreeks in de frontend importeren, zodat je geen API hoeft te bouwen en geen data hoeft op te halen van de frontend. Bij het bouwen voegt Blitz automatisch een API-call toe die de servercode op de server uitvoert. Eigenlijk abstraheert Blitz je API in een compileerstap.
 
-This is a game changer for React app development because it eliminates an entire portion of traditional React app architecture. This means it's easier to learn, faster to develop, and more fun to build things!
+Dit is een doorbraak in de ontwikkeling van React-apps, omdat het een heel deel van de traditionele React-architectuur elimineert. Dit betekent dat het makkelijker te leren, sneller te ontwikkelen, en leuker om dingen te bouwen is!
 
-That said, you can continue doing data fetching via REST or GraphQL like you've always done. Blitz doesn't restrict this in any way.
+Uiteraard kun je blijven gebruik maken REST of GraphQL voor het opvragen van data. Blitz beperkt dit op geen enkele manier.
 
-### 3. Built-in Authentication
+### 3. Ingebouwde authenticatie
 
-Blitz has built-in session management that works with any identity provider, including self-hosted email/password and third party services. Authentication is something that's complicated and difficult to implement correctly. Having it built into Blitz saves you a ton of time and potential security vulnerabilities.
+Blitz heeft een ingebouwd sessiebeheer dat werkt met elke identiteitsprovider, inclusief zelf gehoste e-mail/wachtwoord en diensten van derden. Authenticatie is iets dat ingewikkeld en moeilijk is om correct te implementeren. Doordat dit in Blitz is ingebouwd, bespaar je je een hoop tijd en potentiële beveiligingslekken.
 
-Building an awesome UX with auth in Next.js is very tricky and tedious, but Blitz gives you best-in-class DX for free.
+Het bouwen van een geweldige gebruikerservaring met authenticatie in Next.js is erg lastig en vervelend, maar met Blitz krijgt je gratis de beste ontwikkerlaarservaring in zijn klasse.
 
-### 4. Conventions
 
-Next.js does not have any conventions aside from file-system based routing. This leaves every developer to re-invent the wheel for every project.
+### 4. Conventies
 
-Blitz brings a number of conventions for things like how to organize your files and how to structure your routes. This means you don't waste any time figuring this out for every project. It also greatly benefits the community. Common project structure and architectural patterns allow you to move from Blitz app to Blitz app and immediately feel at home.
+Next.js heeft geen conventies, afgezien van file-system based routing. Dit zorgt ervoor dat de ontwikkelaar telkens het wiel moet heruitvinden bij elk project.
```

Comment:
```suggestion
Next.js heeft geen conventies, afgezien van file-system based routing. Dit zorgt ervoor dat elke ontwikkelaar telkens het wiel moet heruitvinden bij elk project.
```

Reply:
Ik vond die elke hier zo raar, ook in het de Engelse versie. Maar kan het terug toevoegen.